genon/preprocessor/facade/attachment_processor.py

```
@@ -236,6 +245,23 @@ def convert_to_pdf(file_path: str, use_pdf_sdk: bool = True) -> str | None:
+    빌드 시 INSTALL_LIBREOFFICE / INSTALL_RHWP 를 끄거나 PDF SDK 미포함(standard)이면
+    변환 backend 가 0개가 될 수 있다. 가용성 판단 자체가 불가하면(import 실패 등) True 를
+    반환해 기존 동작을 유지한다.
+    """
+    try:
+        from genon.preprocessor.converters.hwp_to_pdf.availability import (
+            libreoffice_available,
+            pdf_sdk_available,
+            rhwp_available,
+        )
+        return bool(pdf_sdk_available() or rhwp_available() or libreoffice_available())
+    except Exception:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)